### DISCUSSION TITLE
Is it ok I'm here?

### DISCUSSION THREAD

**Discussion by KateBogus** [Budibase/budibase#6024] [General]
Hi guys)
I'm a user who tried to fill the survey on survey.ukraine.gov.bg. 
Is it OK I was transfered on GitHub after pressing Help button in the footer of the site? 
I saw your forum even while being unsigned on  GitHub.
And the question: who can help with filling the survey?)
Have a great day everyone

**Comment by R2bEEaton** [comment]
Hello, looks like the owners of the website you were using used Budibase to create it. Budibase or its community cannot support you with a non-technical question about someone else's site.
